Collapsible.js

```diff
@@ -51,6 +51,12 @@ export default class Collapsible extends Component {
     }
   }
 
+  componentDidMount() {
+    if (!this.props.collapsed) {
+      this._toggleCollapsed(this.props.collapsed);
```

Comment:
```suggestion
      this._measureContent((height) => this.state.height.setValue(height));
```
Just sharing other option in case we don't want to show the animation transition initially.
cc: @oblador 

https://github.com/user-attachments/assets/76318485-d2a3-4f0b-97e6-b5856bdbf0c5